### ISSUE TITLE
Bug with movement and executing an action

### ISSUE THREAD

**Issue by ggruts** [otland/forgottenserver#3464]
### Before creating an issue, please ensure:
- [ ] This is a bug in the software that resides in this repository, and not a
      support matter (use https://otland.net/forums/support.16/ for support)
- [ ] This issue is reproducible without changes to the C++ code in this repository

### Expected behaviour
The character moves correctly when using the items and does not have any rollbacks or bugs.

### Actual behaviour
If you run using an item or a spell (from what I tested), the character will be immobile and unable to move forward. You can only get out if you go elsewhere. When you try to move forward, nothing happens. (As I tried to reproduce in the gif below).
I realized that if the speed of the ground is the same along the route, it happens more often. I tested it several times, and it doesn't always happen.

### Reproduction environment
[https://imgur.com/H6qWZhR](url)

**Comment by EPuncker** [comment]
I wasn't able to reproduce the issue with mapclick, the character just stop, but doesn't get stuck

**Comment by ggruts** [comment]
I went to test again and tried to see more details, and I realized that every time it happened, the character didn't hit anywhere. I made another gif trying to demonstrate what I'm saying. I also tested with OTclient 1.0, and it didn't happen with it. (I don’t know if this information is relevant but it’s good to notify).

[https://imgur.com/KMVypju](https://imgur.com/KMVypju)

**Comment by EPuncker** [comment]
> I also tested with OTclient 1.0, and it didn't happen with it. (I don’t know if this information is relevant but it’s good to notify).

yes as you can see in #3411 , it is client sided, on otclient and new tibia clients it works fine, but in old clients it doesn't, and there is nothing we can do 😞

**Comment by ggruts** [comment]
I continued testing and realized that if you lock the fps at 60, it doesn't happen. If it goes beyond that, it starts to happen. The higher the fps above that, the more recurring it happens. I leave this information here in case it is useful sometime. Thanks for listening!

**Comment by Zbizu** [comment]
iirc in cipbia casting spells stopped the movement from click entirely. Will test that later today and edit that comment.
